### PR TITLE
Use ansible way to detect package existence, fixes gluster/gluster-ansible-infra#119

### PR DESCRIPTION
**Pull request by hunter86bg**
Signed-off-by: hunter86bg <hunter86_bg@yahoo.com>

### CODE REVIEW

**Comment by hunter86bg** [comment]
@gobindadas ,

can you take a look ?

**Review by amarts — APPROVED** [review]

File: roles/backend_setup/tasks/main.yml [hunk 1]
```diff
@@ -8,15 +8,22 @@
...
-  command: rpm -q vdsm-python
-  register: rpm_check
-  ignore_errors: yes
+
+# Using Ansible facts to detect vdsm-python package is installed
+- name: Gather the package facts
+  ansible.builtin.package_facts:
+    manager: auto
+
+- name: Print the package facts
```
Comment (mykaul) [inline roles/backend_setup/tasks/main.yml:17]: I'm not clear why you need to do this? (debug leftovers?)

Comment (hunter86bg) [reply roles/backend_setup/tasks/main.yml:17]: The debug will be visible only with '-v' or more , but won't be triggered with regular run.
That's due to 'verbosity: 1'.

Comment (hunter86bg) [reply roles/backend_setup/tasks/main.yml:17]: It shouldn't be a problem at all and it won't slow the code.

Comment (hunter86bg) [reply roles/backend_setup/tasks/main.yml:17]: And future debugging is easierwithout changing the code.

**Pull request merged**